### ISSUE TITLE
Build directory customization fails

### ISSUE THREAD

**Issue by randohinn** [tightenco/jigsaw#298]
My config.php looks like this, since I tried to change the build dir, so Valet would serve the site automagically for testing.

```
<?php

return [
    'production' => false,
    'baseUrl' => '',
    'collections' => [],
    'build' => [
        'destination' => 'public'
    ],
];
```

And when I try to run `vendor/bin/jigsaw build` I now see the following error.

```


PHP Fatal error:  Uncaught Error: Call to undefined method TightenCo\Jigsaw\Console\BuildCommand::confirm() in /home/rando/Documents/Sites/randohinn/vendor/tightenco/jigsaw/src/Console/BuildCommand.php:110
Stack trace:
#0 /home/rando/Documents/Sites/randohinn/vendor/tightenco/jigsaw/src/Console/BuildCommand.php(57): TightenCo\Jigsaw\Console\BuildCommand->confirmDestination()
#1 /home/rando/Documents/Sites/randohinn/vendor/tightenco/jigsaw/src/Console/Command.php(22): TightenCo\Jigsaw\Console\BuildCommand->fire()
#2 /home/rando/Documents/Sites/randohinn/vendor/symfony/console/Command/Command.php(255): TightenCo\Jigsaw\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /home/rando/Documents/Sites/randohinn/vendor/symfony/console/Application.php(901): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 /home/rando/Documents/Sites/randohinn/vend in /home/rando/Documents/Sites/randohinn/vendor/tightenco/jigsaw/src/Console/BuildCommand.php on line 110
```

**Comment by damiani** [comment]
I'll take a look at that error, but you don't need to change your build directory to use Valet; it will automatically serve your file from the default `build_local` directory.

**Comment by randohinn** [comment]
Oh, you're right, it does. Must have been a cache thing the first time around, when I tried, then that caused me to 404 on it.

**Comment by damiani** [comment]
Build directory confirmation issue is now fixed in v1.3.2.
